PyRDF/__init__.py

```diff
@@ -4,7 +4,12 @@
 import logging
 import sys
 
-from PyRDF import Backends
+# TODO
+# No dynamic injection of all submodules for now because that would include
+# also Base, Operation, Node etc. which we think should be private. Once we
+# will change their names with leading underscores those would not clutter the
```

Comment:
We might not even need to change the names of these modules with the underscore to make the private, because in the end the user will never import `PyRDF` (or `DistRDF`, as we could rename it). The user will import stuff in `ROOT.RDF.Distributed`, and that stuff will be injected dynamically from stuff coming from `DistRDF`, which will be the only user-visible interface.